python/MetrologyData.py

```diff
@@ -202,6 +202,22 @@ def resids_boxplot(self, yrange=None, title=None):
         win.set_title(title)
         return win
 
+    # Below applies outlier clipping as in plot_statistics
+    def resids_clip_boxplot(self, nsigma=4, yrange=None, title=None):
+        win = plot.Window()
+	dz = self.resids
```

Comment:
I forgot to comment on this function.  If it isn't actually going to be used, it should probably be deleted from the file.   Given the tab indentation and the comment outside of the function definition, I'm pretty sure I didn't write it.